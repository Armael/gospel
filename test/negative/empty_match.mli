val f : int -> int
(*@ y = f x
    requires match x with *)

(* EXPECTED
   [125] File "empty_match.mli", line 3, characters 26-26:
         Error: Syntax error.
*)
