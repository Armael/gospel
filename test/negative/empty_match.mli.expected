
*******************************
********** Parsed file ********
*******************************
val f : int -> int[@@gospel {| y = f x
    requires match x with |}]
File "empty_match.mli", line 3, characters 26-26:
Error: Syntax error.
